### PR TITLE
DB Connections, Emitters and basic NPC classes

### PR DESCRIPTION
**Pull request by Raserei**

### CODE REVIEW

File: core/src/main/java/utils/math/Rnd.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package utils.math;
+
+import java.util.Random;
+
+/**
+ * Random generator
+ */
+public class Rnd {
+    private static final Random random = new Random();
```
Comment (soloyes) [inline core/src/main/java/utils/math/Rnd.java]: Use SecureRandom please.

File: src/main/java/Main.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 public class Main {
     public static void main(String[] args) {
-        Server server = new Server(9999);
+        Server server = new Server("server.properties");
         try {
             server.start();
```
Comment (soloyes) [inline src/main/java/Main.java:5]: Same?

File: core/src/main/java/skyObjects/Emitters/NPCSkyObjectEmitter.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            float temp = generateTimers.get(type);
+            temp += delta;
+            generateTimers.put(type, temp);
+            if (temp >= Rnd.nextFloat(objType.getMinGenerateTime(), objType.getMaxGenerateTime())) {
+                generateTimers.put(type, 0f);
+                NPCSkyObject npcSkyObject = npcSkyObjectPool.obtain();
+                npcSkyObject.set(reference.get(0),
+                        //Achtung! Not tested!
+                        new Vector2(Rnd.nextFloat(0, worldBounds.getPos().x), Rnd.nextFloat(0, worldBounds.getPos().y)));
+            }
```
Comment (soloyes) [inline core/src/main/java/skyObjects/Emitters/NPCSkyObjectEmitter.java]: Achtung! White the test

**Review by soloyes — APPROVED** [review]

**Review by kirillman1 — APPROVED** [review]

**Review by Naifff — APPROVED** [review]

File: src/main/java/Main.java (hunk 2 above)
Comment (Raserei) [reply src/main/java/Main.java:5]: Same as what?

**Pull request merged**